GliaWidgets/MessageRenderer/ChatMessageRenderer.Interface.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+/// Message renderer for AI custom cards.
+public struct MessageRenderer {
+    var render: (Message) -> UIView?
+}
+
+public extension MessageRenderer {
+    /// Message to render AI custom card view.
+    struct Message {
+        /// Message id.
+        public let id: String
```

Comment:
How about using `Tagged` from Core SDK (it is embedded in Core SDK) to represent Message Id?

Reply:
3e07ec159133ad017cb06c2141cb31e1ff1e431e